### PR TITLE
Many to many table relations

### PR DESCRIPTION
**Pull request by Vgonma**
# Aditions
- Create a table named `vets` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `name`: string
  - `age`: integer
  - `date_of_graduation`: date
- Created a relations table for vets and species called specializations.
- Created a relations table for animals and vets called visits.
- Inserted the data for vets.
- Inserted the data for specialties.
- Inserted the data for visits.
  
# Queries
  - Who was the last animal seen by William Tatcher?
last-animal-William

  - How many different animals did Stephanie Mendez see?
different-animals-Steph

  - List all vets and their specialties, including vets with no specialties.
vets-specs

  - List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
visits-steph-apr-aug

  - What animal has the most visits to vets?
animal-with-most-visits

  - Who was Maisy Smith's first visit?
maisy-first-visit

  - Details for most recent visit: animal information, vet information, and date of visit.
most-recent-visits-details

  - How many visits were with a vet that did not specialize in that animal's species?
non-specialized-visits

  - What specialty should Maisy Smith consider getting? Look for the species she gets the most.
maisy-specialty

### CODE REVIEW

**Review by ShadyShawkat — COMMENTED** [review]
## Hi @Vgonma, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has an empty readme template without any info about the project or authors of the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: data.sql [hunk 1]
```diff
@@ -35,3 +35,149 @@ UPDATE animals SET owner_id = 4 WHERE name LIKE 'Squirtle';
...
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez'), 
+  (SELECT id FROM species WHERE species.name = 'Digimon')
+);
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'Jack Harkness'), 
+  (SELECT id FROM species WHERE species.name = 'Pokemon')
+);
```
Comment (danmainah) [inline data.sql]: - [x] @Vgonma kindly ensure you enter the correct values the specialisation here is digimon
image

File: data.sql [hunk 2]
```diff
@@ -35,3 +35,149 @@ UPDATE animals SET owner_id = 4 WHERE name LIKE 'Squirtle';
...
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Plantmon'),
+  (SELECT id FROM vets WHERE vets.name = 'William Tatcher'),
+  'Aug-10-2020'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Plantmon'),
+  (SELECT id FROM vets WHERE vets.name = 'Maisy Smith'),
+  'Apr-07-2020'
```
Comment (danmainah) [inline data.sql]: - [x] @Vgonma kindly ensure you date is correct, its suppose to be Plantmon visited Maisy Smith on Apr 7th, 2021.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Vgonma , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by codecaiine — APPROVED** [review]
Hi @Vgonma , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great query implementation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**